gammapy/modeling/models/cube.py

```diff
@@ -859,12 +889,17 @@ def evaluate(self):
         """
         value = self.spectral_model(self.energy_center).value
         back_values = self.map.data * value
+        if self.spatial_model is not None:
+            value = self.spatial_model.evaluate_geom(self.map.geom).value
```

Comment:
In general, is it safe to use the `value`? It can silently work, where the unit should fail?

Reply:
We already use `value` for the `spectral_model`.
This way we can use the already existing spatial model without introducing `NormSpatialModel ` for each of them